src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtil.java

```diff
@@ -28,6 +28,30 @@
  */
 public class ParametersValidatorUtil {
 
+    static void checkIsValidString(String stringToValidate,
+                                   String jobParametersName) throws JobParametersInvalidException {
+        checkIsNotNullOrEmptyString(stringToValidate, jobParametersName);
+        checkAsciiString(stringToValidate, jobParametersName);
+        checkSingleLineString(stringToValidate, jobParametersName);
+    }
+
+    static void checkAsciiString(String stringToValidate,
+                                 String jobParametersName) throws JobParametersInvalidException {
+        if (!stringToValidate.matches("\\A\\p{ASCII}*\\z")) {
+            throw new JobParametersInvalidException(
+                    String.format("%s in %s contains non ascii characters", stringToValidate, jobParametersName));
+        }
+    }
+
+    static void checkSingleLineString(String stringToValidate,
+                                      String jobParametersName) throws JobParametersInvalidException {
+        final String NEW_LINE = System.getProperty("line.separator");
```

Comment:
Btw, as I put in the issue...

Spring batch default databases have some hard limits to job parameters.
* Job parameter id can't have more than 100 characters
* Job parameter string value can't exceed 250 characters

While the first is not a problem, The second should be added as a character limitation in the validation to avoid throwing a big nasty error with database table information.